zoo/policies/competition_agent.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import sys
+import os
+import importlib.util
+import shutil
+
+from pathlib import Path, PurePath
+from smarts.core.agent import Agent
+
+
+class CompetitionAgent(Agent):
+    def __init__(self, policy_path):
+        req_file = os.path.join(policy_path, "requirements.txt")
+
+        env_name = Path(policy_path).name  # name of the submission file
+        root_path = Path(__file__).parents[2]  # Smarts main path
+
+        self._policy_dir = policy_path
+        self._comp_env_path = str(os.path.join(root_path, "competition_env"))
+        self._sub_env_path = str(os.path.join(self._comp_env_path, env_name))
+
+        Path.mkdir(Path(self._comp_env_path), exist_ok=True)
+
+        if Path(self._sub_env_path).exists():
+            shutil.rmtree(self._sub_env_path)
+        Path.mkdir(Path(self._sub_env_path))
```

Comment:
This should not be necessary since you can use `parents=True` on line 26.
```suggestion
        if Path(self._sub_env_path).exists():
            shutil.rmtree(self._sub_env_path)
        Path.mkdir(Path(self._sub_env_path), parents=True)
```